src/app/campaign-info/campaign-info.component.ts

```diff
@@ -28,6 +35,9 @@ export class CampaignInfoComponent implements OnInit {
   campaignFinished: boolean;
   campaignRaised: string; // Formatted
   campaignTarget: string; // Formatted
+  donationCount: number;
```

Comment:
I think this could use a docblock. Had a go writing one, the fact that I have an 'or' in it makes me wonder if confusing to have this as a property as well. Maybe there's a better way to document that makes it more intuitive:

```suggestion
  /**
   * May be the count of donations to this campaign, or the to the parent campaign.
   */
  donationCount: number;
```

Reply:
Thanks @bdsl – I've pushed one which I think does more to explain the options, but let me know if you think it should be different.